src/py/flwr/cli/new/templates/app/code/client.baseline.py.tpl

```diff
@@ -0,0 +1,58 @@
+"""$project_name: A Flower $framework_str app."""
```

Comment:
In other templates, we use `/` (same comment for all the other files that are missing the `/`):
```suggestion
"""$project_name: A Flower / $framework_str app."""
```